.github/CONTRIBUTING.md

```diff
@@ -1,6 +1,18 @@
 Contributing
 ============
 
+Choose the right branch
+-----------------------
+
+Before open your pull request, you must determine on which branch you need to
+work.
+
+ * if it contains a bug fix, refactoring or simply some code improvements must
+   be opened against latest minor release branch. If latest stable version is
+   v2.2.3, pull request must be opened starting from branch 2.2.
+
+ * if it contains new features must be opened against master branch.
+
```

Comment:
Can you add a part when you need to merge with master from another branch please?